### PR TITLE
Set embed colour based on activity type

### PR DESCRIPTION
**Pull request by SarenCurrie**
These need to be set in the dictionary. I added the ones we mostly use.

### CODE REVIEW

File: discord/handler.py [hunk 1]
```diff
@@ -148,6 +148,15 @@ def get_token_for_athlete(id):
 
 
 def post_message(body):
+    # See https://developers.strava.com/docs/reference/#api-models-ActivityType
+    activity_colours = {
+        'Run': int('fc4c02', 16),  # orange
```
Comment (Puhapig) [inline discord/handler.py]: I'd prefer this format: `'Run': 0xfc4c02`

File: discord/handler.py [hunk 2]
```diff
@@ -148,6 +148,15 @@ def get_token_for_athlete(id):
...
+        'Ride': int('66c2ff', 16),  # pale blue
+        'Hike': int('008000', 16),  # forest green
+        'RockClimbing': int('ff8000', 16),  # rock colour?
+        'default': int('cc0000', 16)  # red
```
Comment (Puhapig) [inline discord/handler.py]: Also I'm thinking this should be the same as the run colour, since `fc4c02` is the Strava logo colour

Comment (SarenCurrie) [reply discord/handler.py]: This way we know it's an unmapped type.

File: discord/handler.py (hunk 1 above)
Comment (SarenCurrie) [reply discord/handler.py]: Okay. It would require another conversion step though.

Comment (Puhapig) [reply discord/handler.py]: `0x fc4c02` will return a base10 in representation of the hex value, so you should be able to use it without any other conversion.

Comment (SarenCurrie) [reply discord/handler.py]: Ah. Awesome.

File: discord/handler.py (hunk 2 above)
Comment (Puhapig) [reply discord/handler.py]: TBH I'd rather have the default be the same as what it is now, then we can add more types in the future if required.

Comment (SarenCurrie) [reply discord/handler.py]: Ok

**Comment by SarenCurrie** [comment]
@Puhapig please squash when you merge.

**Pull request merged**